File: src/main/java/org/tud/oas/routing/GridFeature.java
package org.tud.oas.routing;

import com.fasterxml.jackson.annotation.JsonProperty;

public class GridFeature {
    @JsonProperty("x")
    public float x;
    @JsonProperty("y")
    public float y;
    @JsonProperty("value")
    public GridValue value;

    public GridFeature() {}
    
    public GridFeature(float x, float y, GridValue value) {
        this.x = x;
        this.y = y;
        this.value = value;
    }
}

File: src/routing/GridFeature.cs
using System;

namespace DVAN.Routing
{
    public class GridFeature
    {
        public float x { get; set; }
        public float y { get; set; }
        public GridValue value { get; set; }

        public GridFeature() { }

        public GridFeature(float x, float y, GridValue value)
        {
            this.x = x;
            this.y = y;
            this.value = value;
        }
    }
}